plugins/orchestrator-backend/src/service/router.ts

```diff
@@ -352,7 +353,10 @@ function setupInternalRoutes(
 
       const decision = await authorize(
         _req,
-        orchestratorWorkflowReadPermission,
+        createPermission({
```

Comment:
how would it work if the user put the global workflow viewing permission but not fine grained ones?

Reply:
what exactly is the global one?

Reply:
a 'global' can be set by the policy author, because now the permission object is matched by a regexp match
so if the policy says 'orchestrator.workflow' then every string that starts with 'orchestrator.workflow' is matched, unless there is a deny rule that matches.

Reply:
we would need to remove the current common permissions, they will not be in use and only leave the ones who are not workflow specific

Reply:
> we would need to remove the current common permissions, they will not be in use and only leave the ones who are not workflow specific

WDYM?